app/models/speedrun_dot_com_category.rb

```diff
@@ -0,0 +1,36 @@
+require 'speedrundotcom'
+
+class SpeedrunDotComCategory < ApplicationRecord
+  belongs_to :category
+
+  def self.from_category!(category)
+    return category.srdc if category.srdc.present?
+
+    if category.game.srdc.nil?
+      srdc_game = SpeedrunDotComGame.from_game!(category.game.srdc.srdc_id).persisted?
```

Comment:
This line is actually throwing an error when running the job.  On line 9 you check if the game is nil, and then if it is you try and get the id from a nil object.

Reply:
Nice catch, think I brainfarted and thought I was calling `SpeedrunDotCom::Game.from_id` here or something. This method needs a `Game`, not a Speedrun.com game ID.